test/src/main/java/org/apache/accumulo/test/functional/CloneTestIT.java

```diff
@@ -352,18 +352,18 @@ public void testCloneWithSplits() throws Exception {
   }
 
   @Test
-  public void testCloneRootTable() {
+  public void testCloneSystemTables() {
     try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
-      assertThrows(AccumuloException.class, () -> client.tableOperations()
-          .clone(AccumuloTable.ROOT.tableName(), "rc1", CloneConfiguration.empty()));
-    }
-  }
-
-  @Test
-  public void testCloneMetadataTable() {
-    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
-      assertThrows(AccumuloException.class, () -> client.tableOperations()
-          .clone(AccumuloTable.METADATA.tableName(), "mc1", CloneConfiguration.empty()));
+      var sysTables = AccumuloTable.values();
+      var tableNames = getUniqueNames(sysTables.length);
+
+      for (int i = 0; i < sysTables.length; i++) {
+        var sysTable = sysTables[i];
+        var cloneTableName = tableNames[i];
+        assertThrows(Exception.class, () -> client.tableOperations().clone(sysTable.tableName(),
+            cloneTableName, CloneConfiguration.empty()));
```

Comment:
Same comment about some throwing AccumuloException and some throwing AccumuloSecurityException